x/delayedack/module.go

```diff
@@ -91,21 +92,25 @@ func (AppModuleBasic) GetQueryCmd() *cobra.Command {
 type AppModule struct {
 	AppModuleBasic
 	keeper keeper.Keeper
+	ibc    *IBCMiddleware // x/delayedack IBC module
```

Comment:
can use the interface
```
type delayedAckIBCModule interface {
	NextIBCMiddleware() porttypes.IBCModule
}
```
as expected by `MsgServer`, instead of concrete struct